lib/rubocop/cop/rails/validation.rb

```diff
@@ -48,6 +48,24 @@ def on_send(node)
         def preferred_method(method)
           WHITELIST[BLACKLIST.index(method.to_sym)]
         end
+
+        def autocorrect(node)
+          _receiver, method_name, *args = *node
+          options = args.delete_if { |arg| arg.type == :sym }.first
+          lambda do |corrector|
+            validate_parameter = method_name.to_s.split('_')[1]
+            corrector.replace(node.loc.selector, 'validates')
+            if options
+              corrector.replace(
+                options.loc.expression,
+                "#{validate_parameter}: { #{options.source} }")
```

Comment:
Not sure what you mean. @jonas054 Any opinion on this? I'm fine even if the autocorrection is not super smart about this.